tests/models/test_ftrl.py

```diff
@@ -122,13 +122,25 @@ def test_ftrl_construct_wrong_double_precision_type():
 
 def test_ftrl_construct_wrong_interactions_type():
     with pytest.raises(TypeError) as e:
-        noop(Ftrl(interactions = 2))
-    assert ("Argument `interactions` in Ftrl() constructor should be an iterable, "
-            "instead got <class 'int'>" == str(e.value))
+        noop(Ftrl(interactions = "C0"))
+    assert ("Argument `interactions` in Ftrl() constructor should be a list or a tuple, "
```

Comment:
Why not use the new `match=` argument to `pytest.raises`?

Reply:
For this I guess we need to make a separate PR by converting all the existing tests to the `match` functionality.

Reply:
As you wish. I was merely thinking that since you're updating the tests anyways, might as well upgrade their syntax too.